### PR TITLE
Release SDK 1.4.0

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,304 @@ Release notes
...
 
+.. _release-1.4.0:
+
+1.4.0 - 2020-08-12
+------------------
+
+Highlights
+~~~~~~~~~~
+
+- DAML on SQL is available as a standalone JAR, `you can find it on
```
Comment (bame-da) [inline docs/source/support/release-notes.rst:17]: This is missing the most recent changes from doc and blog. We are only releasing as Early Access and not advising migration yat.

Comment (cocreature) [reply docs/source/support/release-notes.rst:17]: Updated, the changes were missing from the blog as well which has now been updated.

File: docs/source/support/release-notes.rst [hunk 2]
```diff
@@ -6,6 +6,305 @@ Release notes
...
+~~~~~~~~~~~~~~~~~~~~
+
+Migration from SDK 1.3 to 1.4 should require no action for almost all
+users.
+
+- If you were using the Early Access version of DAML REPL, you need
+  to make a minor adjustment to the CLI parameters to start it.
+- If you are working on application failover, you need to
+  incorporate some API improvements.
+- If you are running the Sandbox with persistence, you are advised
```
Comment (bame-da) [inline docs/source/support/release-notes.rst:51]: This still needs to go.

Comment (cocreature) [reply docs/source/support/release-notes.rst]: :fire:

**Comment by cocreature** [comment]
Manual tests passed on Linux and Windows

**Pull request merged**